src/schema/bytes_options.rs

```diff
@@ -56,13 +71,25 @@ impl BytesOptions {
     }
 }
 
+impl Default for BytesOptions {
+    fn default() -> BytesOptions {
+        BytesOptions {
+            indexed: false,
+            fieldnorms: false,
+            fast: false,
+            stored: false,
+        }
+    }
+}
```

Comment:
why not use the derive? seem to be the same values

Reply:
yes!